boba_community/turing-monsters/contracts/WithOnChainMetaData.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "base64-sol/base64.sol";
+import "./ITuringHelper.sol";
+
+abstract contract WithOnChainMetaData is ERC721 {
+
+    ITuringHelper public turingHelper;
```

Comment:
minor: would it make sense to move turingHelper up a level to NFTMonsters.sol instead?

Reply:
Yeah don't really like it that way either rn. But actually it is already up because NFTMonsters extend the WithOnChainMetadata contract. The same applies to the WithRandomTokenID contract.